rpm/generic/zfs.spec.in

```diff
@@ -165,7 +165,7 @@ This package contains the core ZFS command line utilities.
 %package -n libzpool5
 Summary:        Native ZFS pool library for Linux
 Group:          System Environment/Kernel
-Obsoletes:      libzpool2
```

Comment:
What if someone is upgrading from zfs-0.X to zfs-2.Y?  Should this Obsolete both ```libzpool2``` _and_ ```libzpool4```?

Reply:
I wondered about that myself, but wasn't positive what the expected convention was.  Obsoleting all previous versions makes the most sense to me, I'll go ahead and update this.